src/cloud-element-templates/util/propertyUtil.js

```diff
@@ -658,6 +667,39 @@ export function setPropertyValue(bpmnFactory, commandStack, element, property, v
     });
   }
 
+  // zeebe:calledDecision
+  if (type === ZEEBE_CALLED_DECISION) {
+    let calledDecision = findExtension(element, 'zeebe:CalledDecision');
+    const propertyName = binding.property;
+
+    const properties = {
+      [ propertyName ]: value || ''
+    };
+
+    if (calledDecision) {
+      commands.push({
+        cmd: 'element.updateModdleProperties',
+        context: {
+          element,
+          properties,
+          moddleElement: calledDecision
+        }
+      });
+    } else {
+      calledDecision = createElement('zeebe:CalledDecision', properties, extensionElements, bpmnFactory);
```

Comment:
Thanks to the line coverage, I realized the case that `calledDecision` doesn't exist doesn't really happen. I suppose it's only possible if someone removes the element manually from the xml after applying the template?

Reply:
Hmm perhaps this could happen if we make `calledDecision` property conditional, but that would be a misuse of the templates (either task definition or called decision should be present at time).